crates/editor/src/element.rs

```diff
@@ -3586,7 +3597,8 @@ impl EditorElement {
                         (text_bounds.size.width - size.width) / 2.,
                         text_bounds.size.height - size.height - PADDING_Y,
                     );
-                    element.prepaint_at(text_bounds.origin + offset, window, cx);
+
+                    element.prepaint_at(start_point + offset, window, cx);
```

Comment:
I thought we would need to offset these two popovers as they are supposed to fixed at the top/bottom center